pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -169,6 +167,9 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if (kc.ShutdownGracePeriod.Duration > 0 || kc.ShutdownGracePeriodCriticalPods.Duration > 0) && !localFeatureGate.Enabled(features.GracefulNodeShutdown) {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: specifying shutdownGracePeriod or shutdownGracePeriodCriticalPods requires feature gate GracefulNodeShutdown"))
 	}
+	if kc.PLEGRelistThreshold.Duration <= 0 {
+		allErrors = append(allErrors, fmt.Errorf("invalid configuration: PLEGRelistThreshold must be greater than 0"))
```

Comment:
is it fine to set it to, say, `1s` or should it be proportional to `CFSQuota` and not too small to avoid issues?

Reply:
@matthyx Do you have any suggestions? Thank you much.

Reply:
I am not sure what's a minimum viable duration for this... as for a very small cluster with few pods it could work every few seconds, but for bigger clusters I see danger.
@smarterclayton any idea?

Reply:
I know that this is not directly related to kube controller manager's two flags `--default-not-ready-toleration-seconds` and `--default-unreachable-toleration-seconds` that are by default 300s. In [kubespray](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/kubernetes-reliability.md), to make the cluster more sensitive to node status for pod eviction quickly, we change it to 30s for fast reaction. 

I would suggest making a minimum to be 30 or 20 seconds at least.

Reply:
@pacoxu good idea. I will change the code.

Reply:
@matthyx @pacoxu 30s